locale/hi/LC_MESSAGES/loris.po

```diff
@@ -213,6 +225,9 @@ msgstr "{{totalCount}} में से {{pageCount}} पंक्तिया
 msgid "Maximum rows per page:"
 msgstr "प्रति पृष्ठ अधिकतम पंक्तियाँ:"
 
+msgid "Download Table as CSV"
```

Comment:
Hi @SKADE2303 , we are not longer using "Download Table as CSV" we should always replace by " Download Data as CSV"